.github/workflows/build-website.yml

```diff
@@ -0,0 +1,42 @@
+name: Build Website
+
+on:
+  push:
+    branches:
+      - master
+    paths:
+      - 'docs'
+      - 'website'
+      - 'package.json'
+      - 'packages/**/src'
+
+  pull_request:
+    types: [opened, synchronize, reopened, ready_for_review]
+    branches:
+      - master
+    paths:
+      - 'docs'
+      - 'website'
+      - 'package.json'
+      - 'packages/**/src'
```

Comment:
Probably not needed.

Reply:
Needed because we are building API docs. We want to make sure that the generated links have not broken.